README.md

```diff
@@ -67,4 +67,5 @@ The dependencies json lists specific rules for the validator to follow. When lis
 | `conditionalIgnore` | "If [field1] has a value of [value1], then [field2] is ignored (or shouldn't be provided)" | A dictionary where each key is [field1] and the value is a list of objects that define conditions and [field2] names. If [field2] is in the yaml to validate when the conditions are true, a warning will be given. |
 | `simpleAllowedValues` | "If [field1] has a value of [value1], then [field2] values must be [...]" | A dictionary where each key is a [field1] and the value is an object where each key is a possible value for field 1. Those keys are mapped to objects whose keys are [field2] names with a matching value of an array of possible allowed values for field2 |
 | `deepLinks` | "If [field1] has a value matching one of [a, b, ...] and [field2] has a value matching one of [c, d, ...], then [field3] value must match one of [e, f, ...]" | An object where each key is a shared parent of all fields throughout the rest of the object. For example, fa.fb[].fc is the parent of field1, field2, and field3. Each value contains "sharedParent", "condition" and "requirement".  "condition" is an object where each key-value pair refers to a field (key) and the list of its possible values it must match (value) in order for "requirement" to be required. "requirement" is an object where each key-value pair refers to a field (key) and the list of allowed values (value) for it given the conditions |
+| `fromList` | "[field1] must match one of the values from [field2]" | An object where each value is a [field2] name - a set of keys that will be combined into one list of allowed values for [field1]. The keys of the object are [field1]. [field1]: [field2], where [field1] must match one of the values found when searching through each instance of [field2]|
```

Comment:
I'm having trouble parsing this.  Is there an editing issue / typo somewhere?  If not, I'll re-read a few more times.

Also, I don't understand the name `"fromList"`, but that's just me and it isn't important.

Reply:
I'm really bad at writing these. I wanted to emphasize that in each key-value pair in the fromList object, the value is the key you look at in the yaml to find the list of possible/allowed values for the key in the fromList object. I think the last sentence summarizes it decently, but I agree that the first half can be very confusing. I am open to suggestions.

Reply:
Oh, and as for "fromList", it's because the value of the key has to be a value from a list. It's not a very clever name but I wasn't going to spend too much time deciding what to call it. I'm open to other ideas!

Reply:
"valueMatch"?

Okay, I think I finally understand.  Let me know if this looks correct and clear to you:
- An object in the form [field1]: [field2].  Each value of the object is a field name whose values form the complete list of valid values for the corresponding key, [field1].  The value of [field1] must match one of these values.

Reply:
Done!